pymodels/SI_V24_04/families.py

```diff
@@ -306,6 +306,9 @@ def get_idx(x):
     data['B1B2-1'] = sorted(data['B1']+data['B2'], key=get_idx)
     data['B1B2-2'] = sorted(data['B1']+data['B2'], key=get_idx)
 
+    # all dipoles indices
+    data['BN'] = sorted(data['B1']+data['B2']+data['BC'], key=get_idx)
```

Comment:
Why the choice 'BN' ? 'B' is more intuitive for me and corresponds to string we used in the past to refer to all dipoles.

Reply:
ok.